src/punlist.txt

```diff
@@ -1,2 +1,9 @@
-punnnnnnny
-reeeeee
+In version control, git commits you.
+I typically avoid conversations with robots. They just drone on.
+"beefstew" is not a good password, but "beefStewWithMushroomsAndGravy" might be stroganoff.
+A SQL statement walks into a bar and asks 2 tables if he can join them.
+A computer gets drunk by taking screenshots.
+There are 10 types of people in the world; those who understand binary and those who don't.
+"Computer science" isn't really about computers. And it isn't really a science. But here we are anyways.
+The worst thing you can do at a SQL dinner party is clear the table.
+
```

Comment:
I think the blank line at the end might be a problem, actually.

Reply:
fixed in 8883e2602797976dfdfdab9cdcc4c9dd8cc29e60